lib/display/r_raster.c

```diff
@@ -152,7 +155,28 @@ int D_open_driver(void)
 
     init();
 
-    return 0;
+    /* if GRASS_REGION is not defined, a display module can choose to render
+     * for the entire computational region or wait for the monitor to call it
+     * with GRASS_REGION set to the current display extent (what is shown) in
+     * the monitor; the monitor defines GRASS_REGION to the display extent
```

Comment:
Usage of GRASS_REGION just happens to be the case in d.mon wx0. I don't know much about the display library, but I think the current API allows implementation of a monitor which does not use GRASS_REGION. This makes GRASS_REGION integral part of the monitor system. That's perhaps fine, but it is a change we should be sure about.

Reply:
`GRASS_REGION` is not specific to the `wx0` monitor. `d.mon`'s `render_cmd.py` already uses it to adjust the region:
https://github.com/OSGeo/grass/blob/9c3b67ec82ff1d5db64f6a2baba98390bfe9406c/display/d.mon/render_cmd.py#L115

and I think `GRASS_REGION` was mainly designed for this purpose. Otherwise, the monitor system would have to change the computational region every time it wants to change the display extent. How does `g.gui` do it?

From [variables.html](https://grass.osgeo.org/grass83/manuals/variables.html): _GRASS_REGION ... use is the same as WIND_OVERRIDE. ... This allows programs such as the GUI to run external commands on an alternate region without having to modify the WIND file then change it back afterwards._